packages/kbn-guided-onboarding/src/components/landing_page/guide/guide_card.test.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { cloudMock } from '@kbn/cloud-plugin/public/mocks';
+import { sharePluginMock } from '@kbn/share-plugin/public/mocks';
+import { I18nStart } from '@kbn/core-i18n-browser';
+import { themeServiceMock } from '@kbn/core-theme-browser-mocks';
+import { docLinksServiceMock } from '@kbn/core-doc-links-browser-mocks';
+import { GuideCard } from './guide_card';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+describe('guide cards', () => {
+  test('should render search cards', async () => {
```

Comment:
Ah good point. I think it might make more sense for me to add tests to the getting_started.test.tsx file since setFilteredCards is set up in that file vs in guide_cards.test.tsx as it was in the classic version

Reply:
I agree too, I think the `guide_card.tsx` only needs to be tested with one constant to check that the props are rendered in the right places. The `guide_cards.tsx` doesn't have much logic anymore, so as @rshen91 suggested, it's probably better to test the "filtering" in the component that has that logic.

Reply:
thanks @alisonelizabeth and @yuliacech. I see that the getting_started.test.tsx is already testing the filtering in 'should auto-select guide filter value based on querystring parameter'. I'm thinking that this test is already doing what I was thinking needed to be done 😅.

Reply:
Ok, just to confirm - `getting_started.test.tsx` has the appropriate test coverage therefore we can delete the guide_card(s) test completely and no other changes are required?

Reply:
Yep! I was trying to create new tests but then saw the getting_started.test.tsx already is accomplishing what I was trying to test